packages/morse2ascii/Makefile.patch

```diff
@@ -0,0 +1,10 @@
+--- ./Makefile
++++ ../Makefile
+@@ -1,6 +1,6 @@
+ EXE		= morse2ascii
+ CFLAGS	+= -O2 -s
+-PREFIX	= /usr/local
++PREFIX	= /data/data/com.termux/files/usr
```

Comment:
Might be better to set `PREFIX = $TERMUX_PREFIX`?

Reply:
Or even @TERMUX_PREFIX@ as in other patches :+1: